api/index.js

```diff
@@ -25,6 +25,7 @@ module.exports = async (req, res) => {
     title_color,
     icon_color,
     text_color,
+    not_bold,
```

Comment:
Really confusing boolean. 

Rename it to "text_bold" and keep it enable by default. if user wants to disable bold they can pass text_bold=false

Reply:
@anuraghazra I did it like this since you do not specify default values for the options in the README. I, however, i: agree that `not_bold` is even less clear.